packages/clerk-js/src/core/clerk.ts

```diff
@@ -945,6 +947,12 @@ export default class Clerk implements ClerkInterface {
     this.#authService?.setEnvironment(environment);
   }
 
+  setCountry = (country: CountryIso | null) => {
```

Comment:
⛔ This method needs to be private to Clerk so as not to be exposed as a public API, for example `private __internal_setCountry = (...)`.